1-regen-repos

```diff
@@ -3,6 +3,19 @@
 # Solves 404 error messages when retrieving packages for building the tarballs
 
 . release-common.sh
+
+# make sure osg-koji is available before starting
+check_for_osg_koji
+
+detect_rescue_file
+
+# make sure osg-koji is available before starting
+command -v osg-koji 2>&1 >/dev/null
```

Comment:
`command -v` is broken on bash and doesn't check if the command is actually executable. Try `type osg-koji >/dev/null 2>&1`.